assets/dns/daemonset.yaml

```diff
@@ -44,14 +44,65 @@ spec:
           timeoutSeconds: 5
           successThreshold: 1
           failureThreshold: 5
+      - name: dns-node-resolver
+        # image is set at runtime
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: hosts-file
+          mountPath: /etc/hosts
+        # env NAMESERVER and CLUSTER_DOMAIN are set at runtime
+        command:
+        - /bin/bash
+        - -c
+        - |
+          #!/bin/bash
+          set -euo pipefail
```

Comment:
If we `set -e`, we will exit immediately if the `dig` command below fails; we should probably just not `set -e`.

Reply:
Fixed